redis/src/types.rs

```diff
@@ -2410,6 +2427,110 @@ impl FromRedisValue for () {
 
 macro_rules! from_redis_value_for_tuple {
     () => ();
+    ($name:ident, ) => (
```

Comment:
Improving the documentation at the cost of increasing code complexity isn't really a tradeoff I'm willing to make.

Reply:
If I rework the macros to be less complex, would that be acceptable or are the cfg_attrs also unwantend?:
```rust
#![cfg_attr(docsrs, feature(doc_cfg, rustdoc_internals))]
#![cfg_attr(docsrs, expect(internal_features)]
```